app/views/products/popular.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Dummy Popular Products Page</h2>
+
+<center>
+  <div class="row">
+    <% @products.each do |product| %>
+      <div class="col-sm-6 col-md-3">
+        <div class="thumbnail">
+          <%= link_to (image_tag product.image.url(:thumb), class: 'image'), product %>
+          <div class="caption">
+            <h3><%= product.name %></h3>
+            <h4>Category: <%= product.category %></h4>
+            <p><%= product.description %></p>
+            <p><%= link_to "Read More", product_path(product), class: "btn btn-default", role: "button" %></p>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</center>
```

Comment:
Huh; is this just the same as the products page, but sorted differently? If so, might be more Rails-y to just have this page be `/products?sort_by=popular`, where the `ProductsController#index` method consumes a `sort_by` param to rank the returned `@products` correctly. Does that make sense?

Alternatively, if you want the route to stay `/products/popular`, you could point to the same view from different controller actions ([like this](http://stackoverflow.com/questions/20500316/rails-routing-two-actions-to-point-to-the-same-view))